test/e2e/framework/kubelet_stats.go

```diff
@@ -97,10 +97,13 @@ func getKubeletMetrics(c clientset.Interface, nodeName string) (metrics.KubeletM
 	return kubeletMetrics, nil
 }
 
-// GetKubeletLatencyMetrics gets all latency related kubelet metrics. Note that the KubeletMetrcis
-// passed in should not contain subsystem prefix.
-func GetKubeletLatencyMetrics(ms metrics.KubeletMetrics) KubeletLatencyMetrics {
-	latencyMethods := sets.NewString(
+// GetDefaultKubeletLatencyMetrics calls GetKubeletLatencyMetrics with a set of default metricNames
+// identifying common latency metrics.
+// Note that the KubeletMetrics passed in should not contain subsystem prefix.
+// TODO(mtaufen): It looks like some latency metrics have been added since latencyMethods in this function was updated,
+// like kubeletmetrics.EvictionStatsAgeKey. Should we add those here as well?
+func GetDefaultKubeletLatencyMetrics(ms metrics.KubeletMetrics) KubeletLatencyMetrics {
+	latencyMetricNames := sets.NewString(
```

Comment:
looks like this is a const.

Reply:
You can't use `const` with function calls (e.g. any constructor).
I don't see a reason to lift the scope, since nothing else needs this.